tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
+  "compilerOptions": {
+    "target": "esnext",
+    "module": "esnext",
+    "strict": true,
+    "jsx": "preserve",
+    "importHelpers": true,
+    "moduleResolution": "node",
+    "esModuleInterop": true,
+    "allowSyntheticDefaultImports": true,
+    "sourceMap": true,
+    "baseUrl": ".",
+    "types": [
+      "webpack-env",
+      "jest"
+    ],
+    "paths": {
+      "@/*": ["src/*"],
+      "@types": ["types"]
```

Comment:
I am worried that `@types` may cause confusion for people when they encounter `@type/*`packages. It might be clearer to move `/types/index.ts` to `/src/types.ts` - then you can include it with `@/types`. However, if this is standard practice in vue projects, go ahead with it

Reply:
I don't think it's standard practice, but I saw the pattern in a project linked from the docs. Won't matter after the TS is gone. 😁